webapp/src/main/java/life/qbic/views/login/LoginHandler.java

```diff
@@ -37,25 +41,44 @@ public void handle(LoginLayout loginView) {
       registeredLoginView = loginView;
     }
     initFields();
-    resetMessages();
     addListener();
   }
+
   private void initFields() {
-    registeredLoginView.password.setHelperText("A password must be at least 8 characters");
-    registeredLoginView.password.setPattern(".{8,}");
-    registeredLoginView.password.setErrorMessage("Invalid Password");
-    registeredLoginView.email.setErrorMessage("Invalid Email");
+    resetErrorMessages();
+    resetInformationMessages();
+  }
+
+  private void resetErrorMessages() {
+    showDefaultError();
+    hideError();
+  }
+
+  private void resetInformationMessages() {
+    registeredLoginView.resetInformation();
+  }
+
+  private void hideError() {
+    registeredLoginView.hideError();
+  }
+
+  private void showDefaultError() {
+    registeredLoginView.showError(INCORRECT_USERNAME_OR_PASSWORD);
+  }
```

Comment:
As I mentioned I would like to keep the default text in there to prevent random errors to show up. I like your suggestion however I would add another method like you suggested when we can differentiate between the errors. The method `showDefaultError()` can exist as a fallback.

Reply:
As the errors are never shown and hidden but added and removed this is not valid anymore